### ISSUE TITLE
Why not apply Softmax in the last layer?

### ISSUE THREAD

**Issue by HesperX** [zylo117/Yet-Another-EfficientDet-Pytorch#413]
Hello, thank you for this great work!
I notice that you apply the sigmoid function instead of softmax in the last layer of classification network.
I assume that each object can only belong to a unique category so maybe it is more suitable to apply softmax? I'd like to ask the reason for your choice. And I want to know whether the official code applies sigmoid too?

Thank you very much.

**Comment by zylo117** [comment]
confidence could be much lower if using softmax as num_classes gets greater

**Comment by HesperX** [comment]
Thank you for the reply.
But may the sigmoid lead to false postive? I tried your model. In some test samples, some categories have similar confidence such as skateboard and snowboard. Did you test with softmax layer? Or is it a typical choice for coco dataset?

**Comment by zylo117** [comment]
I think if there are false positve issues when using sigmoid, changing to softmax will also lead to the same result, like 0.49 and 0.51.

**Comment by HesperX** [comment]
OK. Thank you very much.
